packages/plugin-core/src/test/suite-integ/ShowPreview.test.ts

```diff
@@ -0,0 +1,90 @@
+import { DendronWebViewKey } from "@dendronhq/common-all";
+import { ENGINE_HOOKS } from "@dendronhq/engine-test-utils";
+import { test } from "mocha";
+import sinon from "sinon";
+import { ExtensionContext } from "vscode";
+import { ShowPreviewCommand } from "../../commands/ShowPreview";
+import { VSCodeUtils } from "../../utils";
+import { MarkdownUtils } from "../../utils/md";
+import { DendronWorkspace } from "../../workspace";
+import { expect } from "../testUtilsv2";
+import { runLegacyMultiWorkspaceTest, setupBeforeAfter, withConfig } from "../testUtilsV3";
+
+
+suite("ShowPreview", function () {
+  let ctx: ExtensionContext;
+  ctx = setupBeforeAfter(this);
+
+	test("ok: show previewv2 by default", (done) => {
+
+		runLegacyMultiWorkspaceTest({
+			ctx,
+			preSetupHook: ENGINE_HOOKS.setupBasic,
+			onInit: async ({ engine }) => {
+				const note = engine.notes["foo"];
+				await VSCodeUtils.openNote(note)
+				await new ShowPreviewCommand().execute();
+				const preview = DendronWorkspace.instance().getWebView(DendronWebViewKey.NOTE_PREVIEW);
+				expect(preview?.visible).toBeTruthy();
+				done();
+			},
+		});
+
+	});
+
+	test("ok: prompt install for legacy preview when missing", (done) => {
+
+		runLegacyMultiWorkspaceTest({
+			ctx,
+			preSetupHook: ENGINE_HOOKS.setupBasic,
+			onInit: async ({ engine, wsRoot }) => {
+				withConfig(
+					(config) => {
+						config.dev = {
+							enablePreviewV2: false
+						}
+						return config;
+					},
+					{ wsRoot }
+				);
+				const note = engine.notes["foo"];
+				await VSCodeUtils.openNote(note)
+				sinon.stub(MarkdownUtils, "hasLegacyPreview").returns(false)
```

Comment:
I still haven't been able to figure out why, but I don't think `sinon.restore()` (default sandbox restore) will work on the Windows test pass and may cause issues since there are multiple instances of the same stub in this file.  Might be good to explicitly restore the stub object, i.e.

```typescript
const blah = sinon.stub(...)
...
blah.restore()
```